src/views/templates/details/tabs/details/components/Annotations.tsx

```diff
@@ -12,6 +12,8 @@ import {
   Popover,
 } from '@patternfly/react-core';
 
+import 'src/views/templates/details/tabs/details/TemplateDetailsPage.scss';
```

Comment:
the annotations component should not need this scss import,
this scss should be called from TemplateDetailsPage.ts
if you need it here, you need to look why, and adress the issue some other way